Documentation/cold-staking-setup.md

```diff
@@ -0,0 +1,65 @@
+## COLD STAKING SETUP INSTRUCTIONS
+
+**What you will need** 
+
+ - **Hot wallet** - this will be used for staking online, you can skip this if you are using _www.trustaking.com_
+ - **Cold wallet** - this will hold your funds offline
+   
+_Please note. It's important that both wallets are running on separate full nodes._
+
+---
+****Hot Wallet****
+
+_If you are using www.trustaking.com, your **coldstakinghotaddress** will be provided for you on the website and you can skip this section._
+
+1. Convert your Hot wallet to be enable cold staking by using the the `cold-staking-account` API method with isColdWalletAccount set to "false"
+2. Get your Hot wallets coldstakinghotaddress using by using the `cold-staking-address` API method with "IsColdWalletAddress" set to "false"
+3. Then start the node staking with the Hot wallet from the command line or config file.
+ 
+ ---
+****Cold wallet****
+
+_If you are using www.trustaking.com, we provide a command to simplify this process._
+
+1. Fund the Cold wallet with coins that you eventually want to stake, these coins should go into the standard "account 0".
```

Comment:
In this case (see below) you seem to be funding the setup transaction from the hot wallet, so perhaps this should read "Fund the Hot wallet with coins..."

Reply:
Updated to reflect that the funding is from the cold wallet